### PR TITLE
[WIP] Document Feature API endpoints

### PR DESCRIPTION
**Pull request by ConnorChristensen**

### CODE REVIEW

File: api/app/serializers/v1/feature_serializer.rb [hunk 1]
```diff
@@ -5,32 +5,31 @@ class FeatureSerializer < ManifoldSerializer
...
-    typed_attribute :background_styles, Types::Hash
+    typed_attribute :header, Types::String
+    typed_attribute :header_formatted, Types::String.meta(read_only: true)
+    typed_attribute :subheader, Types::String.optional
+    typed_attribute :subheader_formatted, Types::String.optional.meta(read_only: true)
+    typed_attribute :body, Types::String
+    typed_attribute :body_formatted, Types::String.meta(read_only: true)
+    typed_attribute :link_text, Types::String
+    typed_attribute :link_url, Types::Serializer::URL
+    typed_attribute :link_target, Types::String.optional # TODO: Add description for what this does
```
Comment (ConnorChristensen) [inline api/app/serializers/v1/feature_serializer.rb]: I do not think I understand how a link target differs from a link url.

File: api/app/serializers/v1/feature_serializer.rb [hunk 2]
```diff
@@ -5,32 +5,31 @@ class FeatureSerializer < ManifoldSerializer
...
+    typed_attribute :style, Types::String.enum("dark", "light")
+    typed_attribute :hidden, Types::Bool
+    typed_attribute :background_color, Types::String.optional.meta(description: "Any valid CSS color (eg. 'blue' or '#e3e3e3'")
+    typed_attribute :foreground_color, Types::String.optional.meta(description: "Any valid CSS color (eg. 'blue' or '#e3e3e3'")
+    typed_attribute :header_color, Types::String.optional.meta(description: "Any valid CSS color (eg. 'blue' or '#e3e3e3'")
+    typed_attribute :layout, Types::String.optional # TODO: Check type for this. Could not find it used on the front end
```
Comment (ConnorChristensen) [inline api/app/serializers/v1/feature_serializer.rb]: Is this used anymore? If so, what does it do and what type is it?

File: api/app/serializers/v1/feature_serializer.rb (hunk 1 above)
Comment (zdavis) [reply api/app/serializers/v1/feature_serializer.rb]: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#target

File: api/app/serializers/v1/feature_serializer.rb (hunk 2 above)
Comment (zdavis) [reply api/app/serializers/v1/feature_serializer.rb]: Not sure. Just leave it as a string and we'll come back to it later.

File: api/app/serializers/v1/feature_serializer.rb (hunk 1 above)
Comment (ConnorChristensen) [reply api/app/serializers/v1/feature_serializer.rb]: Thank you. I added that to the description of the attribute.

**Pull request merged**